### ISSUE TITLE
Upload all images linked in the curriculum to the curriculum repo

### ISSUE THREAD

**Issue by wise-king-sullyman** [TheOdinProject/curriculum#22433]
### Background
It's been discovered that our current method for linking images in the curriculum has issues. We need to change it.

### What needs to be done
We need to upload all of our images to the curriculum repo itself, then update the links to use a free CDN (Content Delivery Network) that we've selected. This will need to be done in two parts and the PR associated with *this* issue should **only** add the images to the repo and **not** change any of the links. I will create a separate issue for changing the links to the images once this image upload PR is merged and I've written up some suitable documentation on how to do so. 

Please see issues #22414 and #22415 and their associated PRs for the folder structure and file naming patterns to follow. If you're unclear on what exactly the patterns are please ask. 

### Completion Criteria
- [ ] The images from all external image links (excluding those already replaced in the issues referenced above) in the curriculum  have been uploaded to the repo
- [ ] The folder structure and file naming patterns from the issues referenced above are followed to maintain consistency

### Questions
If you have any questions please comment them here

**Comment by xiaozhong21** [comment]
I'm new to TOP contribution, and I'll be happy to work on this one!

**Comment by wise-king-sullyman** [comment]
Sounds good, thanks for taking it on @xiaozhong21!
